uaclient/status.py

```diff
@@ -678,5 +676,18 @@ def format_json_status(status: "Dict[str, Any]") -> str:
 
     # We don't need the origin info in the json output
     status.pop("origin", "")
+    return status
+
+
+def format_json_status(status: "Dict[str, Any]") -> str:
+    from uaclient.util import DatetimeAwareJSONEncoder
+
+    return json.dumps(
+        _format_status_output(status), cls=DatetimeAwareJSONEncoder
+    )
+
+
+def format_yaml_status(status: "Dict[str, Any]") -> str:
+    import yaml
```

Comment:
We need to make this import here because on `setup.py` we indirectly import this module before installing the package dependencies, which breaks the build if the import is made on the beginning of the file